### ISSUE TITLE
Bug with Textarea autosizing feature when livewire update is triggered on filament page

### ISSUE THREAD

**Issue by tjodalv** [laravel-filament/filament#13891]
### Package

filament/filament

### Package Version

v3.2.97

### Laravel Version

v10.48.19

### Livewire Version

v3.5.4

### PHP Version

PHP 8.2.17

### Problem description

On edit or create form when you have `Textarea` component and `autosize()` is enabled on first page load (in case of edit page) component behave ok. But if any action on that page triggers change (livewire update) then textarea component will shrink to default height. If you then type again something in the textarea component then it will resize properly.

### Expected behavior

If autosize is enabled on the textarea component I want to be able to see whole text in component always without the scrollbar.

### Steps to reproduce

Add `Textarea` component with `autosize()` option enabled in any form. Then add `Toggle` component with `live()` option enabled so that something can trigger change on the edit page.

```php
\Filament\Forms\Components\Toggle::make('check_me')->live(),
\Filament\Forms\Components\Textarea::make('description')->autosize()
```

Add few lines of text in the Texarea component. Then go to edit page. Everything should be fine. All lines in the text area should be visible. Now, trigger the changes with Toggle component and text area component will shrink to some default height.

### Reproduction repository (issue will be closed if this is not valid)

https://github.com/filamentphp/filament

**Comment by ariStridely** [comment]
Encountered the same problem which can be reproduced by adding this simple snippet to a Page:

```php
public function table(Table $table): Table
{
    return $table
        // ...
        ->actions([
            TableAction::make('some action')
                ->button()
                ->form(
                    function () {
                        return [
                            Textarea::make('some action')
                                ->default(str_repeat('a', 1000))
                                ->hintAction(
                                    FormAction::make('trigger livewire request')
                                )
                                ->autosize()
                        ];
                    }
                )
        ]);
}
```

![chrome-capture-2024-7-28](https://github.com/user-attachments/assets/e1873bdb-7a99-44f6-aebc-d91b93f1d8ad)

**Comment by tjodalv** [comment]
That bug is fixed in version 3.2.98. Just update filament package to the latest version.
